README.md

```diff
@@ -36,6 +36,11 @@ jobs:
 ```
 4. If, for some reason, you need to change the location of the SSH agent socket, you can use the `ssh-auth-sock` input to provide a path.
 
+If you need to use multiple SSH keys (for example because you're using deployment keys which belong to one repository each), you can put all those keys into the secret value one after another.
+This action will add all the keys to the `ssh-agent` one-by-one. `ssh` will then automatically figure out the right keys for each connection.
```

Comment:
I feel like this needs some disclaimer that if you add too many keys, ssh-agent will feed all of them to ssh servers in the order they are added. Most servers have a limit of attempts to authenticate with a key. For example, if the server has a limit of 5 keys and the correct one is number 6 in the sequence, ssh client will use 5 first keys and then ssh server will interrupt the auth flow and that right one won't even be tried out.